p2p/tests/integration/p2p.rs

```diff
@@ -36,7 +37,9 @@ fn setup_logger() {
         let log_config = Configuration {
             max_log_level: Level::TRACE.into(),
             compact_mode: false,
-            ..Configuration::default()
+            ..ConfigurationProxy::default()
+                .build()
+                .expect("Default logger config failed to build. This is a programmer error")
```

Comment:
We have combination of `default -> build -> expect` all over the place.
I wonder if we can add method like `ConfigurationProxy::build_default() -> Configuration` which would return `Configuration` object directly.

Reply:
Like this one, a readability boost. Will do.

Reply:
On second thought, not really feasible in any of the places. E.g. logger configs in tests are supplied traces externally, and only some of the fields are default; queue configs also overload some of its default values in all of their tests.